src/model/openthaigpt_pretraining_model/datasets/__init__.py

```diff
@@ -3,7 +3,7 @@
 from typing import Union
 
 
-def get_dataset(
+def get_dataset_for_lion(
```

Comment:
You should copy this whole function into lion experiment scrpits (Note: We will refactor it later.)

Reply:
done